builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -124,6 +136,12 @@ func resourceVSphereVirtualMachine() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"linkedClone": &schema.Schema{
```

Comment:
@aheeren I think this will be asked to change since it doesn't meet naming convention standards. Probably asked for `linked_clone` and this may conflict with other PRs that are currently out there.